src/main/resources/META-INF/rewrite/spring-security-60.yml

```diff
@@ -33,3 +33,21 @@ recipeList:
   - org.openrewrite.java.spring.security6.UseSha256InRememberMe
   - org.openrewrite.java.spring.security6.PropagateAuthenticationServiceExceptions
   - org.openrewrite.java.spring.security6.RequireExplicitSavingOfSecurityContextRepository
+---
+########################################################################################################################
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.security6.RemoveUseAuthorizationManager
```

Comment:
Should this be included as part of  the larger spring-security migration?

Reply:
Yes, going to add this to the spring-security migration recipe list.

Reply:
added to the spring security 6 recipe list